src/ucs/sys/sys.h

```diff
@@ -362,6 +362,45 @@ void *ucs_sys_realloc(void *old_ptr, size_t old_length, size_t new_length);
  */
 void ucs_sys_free(void *ptr, size_t length);
 
+
+/**
```

Comment:
put all code on new files:
ucs/sys/topo.h
ucs/sys/topo.c

function names:
ucs_topo_find_device_by_pci_id(bus_id)->sys_device*
ucs_topo_find_device_by_name(name)->sys_device*
ucs_topo_get_distance(device1,device2, &result). while "result" is struct type with the fields: { latency, bandwidth }